arangod/Aql/EnumerateCollectionExecutor.cpp

```diff
@@ -174,14 +174,14 @@ std::tuple<ExecutionState, EnumerateCollectionStats, size_t> EnumerateCollection
 
     if (_state == ExecutionState::WAITING) {
       return std::make_tuple<ExecutionState, EnumerateCollectionStats, size_t>( // tupple, cannot use initializer list due to build failure
-        ExecutionState(_state), std::move(stats), 0 // rvalue for first argument and std::move(...) required tmatch constructor signature
```

Comment:
isn't `ExecutionState(_state)` already an rvalue ?

Reply:
I would think so too, but MSVC was complaining with:
cannot convert argument 1 from 'arangodb::aql::ExecutionState' to 'arangodb::aql::ExecutionState &&'